src/runtime/providers/netlifyImageCdn.ts

```diff
@@ -0,0 +1,53 @@
+import { encodeQueryItem } from 'ufo'
+import type { ProviderGetImage } from '../../types'
+import { createOperationsGenerator } from '#image'
+
+// https://docs.netlify.com/image-cdn/overview/
+export const operationsGenerator = createOperationsGenerator({
+  keyMap: {
+    width: 'w',
+    height: 'h',
+    format: 'fm',
+    quality: 'q',
+    position: 'position',
+    fit: 'fit'
+  },
+  valueMap: {
+    fit: {
+      fill: 'fill',
+      cover: 'cover',
+      contain: 'contain'
+    },
+    format: {
+      avif: 'avif',
+      gif: 'gif',
+      jpg: 'jpg',
+      png: 'png',
+      webp: 'webp'
+    },
+    position: {
+      top: 'top',
+      right: 'right',
+      bottom: 'bottom',
+      left: 'left',
+      center: 'center'
+    }
+  },
+  joinWith: '&',
+  formatter: (key, value) => encodeQueryItem(key, value)
```

Comment:
@ascorbic Just updated this but want to confirm - is it correct that we should encode query items, e.g. `?url=%2Fimage.jpg` instead of `?url=/image.jpg` or do you prefer to keep the slashes unencoded?

Reply:
@danielroe It looks like either works, but I reckon encoding is safer